paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt

```diff
@@ -39,11 +41,22 @@ public class AccessibilityRenderExtension : RenderExtension {
   override fun renderView(
     contentView: View
   ): View {
+    // Window Manager needed to access accessibility elements for views that draw to other windows.
+    val windowManager = contentView.context.getSystemService(WindowManager::class.java)
+
     return LinearLayout(contentView.context).apply {
       orientation = LinearLayout.HORIZONTAL
       weightSum = 2f
       layoutParams = ViewGroup.LayoutParams(MATCH_PARENT, MATCH_PARENT)
 
+      viewTreeObserver.addOnGlobalLayoutListener {
+        // Window manager is rendered at the root of the view hierarchy, rendering it full width.
+        // We need to restrict it when taking accessibility snapshots.
+        (windowManager as WindowManagerImpl).currentRootView?.let {
+          layoutParams = FrameLayout.LayoutParams(contentView.measuredWidth, MATCH_PARENT, Gravity.START)
```

Comment:
interesting, is this how layoutlib does it?

Reply:
question: will this layout change affect subsequent non-accessibility tests, i.e., do we need to revert back to full width in some sort of shutdown?

Reply:
> interesting, is this how layoutlib does it?
No this is specifically to get AccessibilityRenderExtension to render as expected.

> question: will this layout change affect subsequent non-accessibility tests, i.e., do we need to revert back to full width in some sort of shutdown?

Hmm, we can I guess. I hadn't thought about that. Whats the use case where this is needed?

Reply:
Oh, I don't know if this is *needed* per se, thus the question.  

I'm thinking of scenarios where you run two tests, an accessibility test and a non-accessibility test, in that order, and whether the second test may render incorrectly.

Actually, since this is mutating the currentRootView, which is presumably detached/re-attached to between tests, then this shouldn't be a problem?